package.json

```diff
@@ -10,22 +10,30 @@
     "start": "node dist/www/index.js",
     "lint": "npx eslint '**/*.ts'",
     "lint:fix": "yarn run lint --fix",
+    "test": "npx jest",
     "no-lock": "scripts/noPackageLock.sh"
   },
   "devDependencies": {
+    "@types/cors": "^2.8.12",
     "@types/express": "^4.17.13",
+    "@types/jest": "^27.0.3",
     "@types/node": "^14.14.35",
+    "@types/node-fetch": "2.5.12",
     "@types/oracledb": "^5.2.1",
     "@typescript-eslint/eslint-plugin": "^4.19.0",
     "@typescript-eslint/parser": "^4.19.0",
     "eslint": "^7.22.0",
     "eslint-plugin-import": "^2.22.1",
+    "jest": "^27.4.5",
     "nodemon": "^2.0.15",
+    "ts-jest": "^27.1.1",
     "ts-node": "^10.1.0",
     "typescript": "^4.2.3"
   },
   "dependencies": {
+    "cors": "^2.8.5",
     "express": "^4.17.1",
+    "node-fetch": "2.6.6",
```

Comment:
using `2.6` bc `3.x` requires esmodules which is a nightmare with some packages we have.